### PR TITLE
make UBNT image names consistent

### PR DESCRIPTION
**Pull request by oakey-dev**
as there is often the question if this version is supported as well

### CODE REVIEW

File: targets/ar71xx-generic/profiles.mk [hunk 1]
```diff
@@ -179,12 +179,26 @@ $(eval $(call GluonModelAlias,UBNT,ubiquiti-nanostation-m,ubiquiti-nanostation-m
...
+
 $(eval $(call GluonModel,UBNT,ubnt-unifi,ubiquiti-unifi))
+$(eval $(call GluonModelAlias,UBNT,ubiquiti-unifi,ubiquiti-unifi-ap))
+$(eval $(call GluonModelAlias,UBNT,ubiquiti-unifi,ubiquiti-unifi-ap-lr))
+
+$(eval $(call GluonModel,UBNT,ubnt-uap-pro,ubiquiti-unifi-ap-pro))
 $(eval $(call GluonModel,UBNT,ubnt-unifi-outdoor,ubiquiti-unifiap-outdoor))
+$(eval $(call GluonModelAlias,UBNT,ubiquiti-unifiap-outdoor,ubiquiti-unifi-ap-outdoor))
 $(eval $(call GluonModel,UBNT,ubnt-unifi-outdoor-plus,ubiquiti-unifiap-outdoor+))
+$(eval $(call GluonModelAlias,UBNT,ubiquiti-unifiap-outdoor+,ubiquiti-unifi-ap-outdoor+))
```
Comment (NeoRaider) [inline targets/ar71xx-generic/profiles.mk]: These last two are just adding a `-`, please remove them. The rest looks good (but we should probably try to make the model names in upsteam OpenWrt/LEDE more consistent with regard to unifi/unifiap/unifi-ap/uap naming).

**Comment by Adorfer** [comment]
Side-note to AP-LR: Can anybody please verify if the LUCI-setup-mode (via pushbutton after having started) works on AP-LR. 
Possibly it's hardware fault, but i can't verfify, i just have one to test. and Pushbutton-TFTP-Boot works.

**Comment by rotanid** [comment]
@oakey-dev are you planning to fix the problem that @NeoRaider pointed out in your Pull Request?

**Comment by rotanid** [comment]
@oakey-dev i guess this PR would have to wait until the LEDE-based gluon branch is ready and updated afterwards, including upstreaming. ( cc @NeoRaider )

**Pull request merged**